src/commands/transaction/aggregatekeylinks.ts

```diff
@@ -0,0 +1,197 @@
+/*
+ *
+ * Copyright 2018-present NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import { command, metadata, option } from 'clime';
+import { Deadline, PlainMessage, PublicAccount, TransactionType, TransferTransaction } from 'symbol-sdk';
+import { AnnounceTransactionsCommand } from '../../interfaces/announce.transactions.command';
+import { AnnounceTransactionsOptions } from '../../interfaces/announce.transactions.options';
+import { MultisigAccount } from '../../models/multisig.types';
+import { AggregateTypeResolver } from '../../resolvers/aggregateType.resolver';
+import { MaxFeeResolver } from '../../resolvers/maxFee.resolver';
+import { MosaicsResolver } from '../../resolvers/mosaic.resolver';
+import { PasswordResolver } from '../../resolvers/password.resolver';
+import { PublicKeyResolver } from '../../resolvers/publicKey.resolver';
+import { AccountService } from '../../services/account.service';
+import { TransactionSignatureOptions } from '../../services/transaction.signature.service';
+import AccountKeyLinkCommand from './accountkeylink';
+import VotingKeyLinkCommand from './votingkeylink';
+import VrfKeyLinkCommand from './vrfkeylink';
+
+export class CommandOptions extends AnnounceTransactionsOptions {
+    @option({
+        description: 'Linked Remote Account Public Key.',
+    })
+    remotePublicKey: string;
+
+    @option({
+        description: 'Remote Account Key Link action (Link, Unlink).',
+    })
+    remoteLinkAction: string;
+
+    @option({
+        description: 'Linked Voting Account Public Key.',
+    })
+    votingPublicKey: string;
+
+    @option({
+        description: 'Voting Key Link Start Point.',
+    })
+    votingStartPoint: string;
+
+    @option({
+        description: 'Voting Key Link End Point.',
```

Comment:
Point or Epoch?

Reply:
It is stated as Point in the votingKeyLink subcommand
https://github.com/nemtech/symbol-cli/blob/main/src/commands/transaction/votingkeylink.ts

Reply:
OK, I don't understand yet what either means anyway.